src/Moose-Core/MooseModel.class.st

```diff
@@ -470,6 +501,26 @@ MooseModel >> importFrom: aFMModel named: aString [
 	self name: aString
 ]
 
+{ #category : #actions }
+MooseModel >> importFromJSONStream: aStream [
+	"Benchmarks
+	Time millisecondsToRun: [ MooseModel new importFromMSEStream: (StandardFileStream readOnlyFileNamed: 'network3.mse') ].
+	Time millisecondsToRun: [ MooseModel new importFromMSEStream: (StandardFileStream readOnlyFileNamed: 'moose.mse') ]. 178163 simon.denier 9/11/2009 12:29
+	 23678 -> simon.denier 9/11/2009 11:36 - initial run
+ 	 21551 -> simon.denier 9/11/2009 12:18 - IdentityHashSet (reverted)
+	 17560 -> simon_denier 9/21/2009 22:34 - removing metrics from MSE
+	"
+
+	^ self importFrom: (self class importFromJSON: aStream withMetamodel: self metamodel) named: (aStream localName removeSuffix: '.mse')
```

Comment:
I thing you forgot this one: '.mse' -> '.json'

Reply:
Done

Reply:
I have also changed the comment